AxonDeepSeg/visualization/get_masks.py

```diff
@@ -17,17 +21,16 @@ def get_masks(path_prediction):
     myelin_prediction = prediction > 100
     myelin_prediction = myelin_prediction ^ axon_prediction
 
-    # get main path
-    path_folder, file_name = os.path.split(path_prediction)
-
-    tmp_path = path_prediction.split('_seg-axonmyelin')
-
-    # save the masks
-    if tmp_path[0].endswith('.png'):
-        tmp_path[0] = os.path.splitext(tmp_path[0])[0]
+    # We want to keep the filename path up to the '_seg-axonmyelin' part
+    path = path_prediction.parent
+    broken_filename = path_prediction.name.split('_seg-axonmyelin')[0]
```

Comment:
I think we should use another word for the prefix in `broken_filename`; I though there was something wrong with it. Maybe `filename_prefix`, `split_filename`, `common_prefix`, etc?